jobs/scrape_archives.py

```diff
@@ -146,9 +153,11 @@ def archive_url(product, version=None, platform=None, locale=None):
     info = client.server_info()
     batch_size = info["settings"]["batch_max_requests"]
     while True:
+        # Gather a batch of records.
         records = [await queue.get()]
         while not queue.empty() and len(records) < batch_size:
             records.append(await queue.get())
+
         task = loop.run_in_executor(executor, publish_records, client, records)
         # Counting tasks done allows us to wait for the queue with queue.join()
         task.add_done_callback(lambda fut: queue.task_done())
```

Comment:
Maybe here we should call `task_done()`  as many time as created records in batch?